scripts/docker-entrypoint.sh

```
@@ -19,13 +21,79 @@ set -e
+    "R2_SECRET_KEY",
+    "JUICEFS_META_URL_TEMPLATE",
+    "JFS_ENCRYPTION_KEY",
+)
+with open(sys.argv[1], "w", encoding="utf-8") as handle:
+    for key in _JFS_KEYS:
+        value = os.environ.get(key)
+        if value:
+            handle.write(f"export {key}={shlex.quote(value)}\n")
+PY
+    then
+        . "$_jfs_env_file"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)